src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -126,6 +128,21 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return across.rateModel.parseAndReturnRateModelFromString(config.routeRateModel[route]);
   }
 
+  /**
+   * Resolve the implied set of chain ID indices based on the chain ID of the ConfigStore.
+   * @param ConfigStore chain ID.
+   * @dev If the resolved chain ID is part of the default set, assume the protocol defaults.
+   *      Otherwise, assume this is a test deployment with a lone chain ID.
+   * @dev The protocol defaults are [1, 10, 137, 288, 42161](outlined in UMIP-157).
+   * @dev chainId is marked optional to appease tsc. It must always be passed in.
+   */
+  protected implicitChainIdIndices(chainId?: number): number[] {
+    assert(isDefined(chainId), `ConfigStoreClient used before update`);
+    return PROTOCOL_DEFAULT_CHAIN_ID_INDICES[0] === chainId
```

Comment:
nb. This imposes an implicit assumption that the ConfigStore contact was deployed to the first chain in the array, which is also implicitly the Hub chain. FWIW I think that's OK.